judges/type-was-attached/type-was-attached.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 Zerocracy
+# SPDX-License-Identifier: MIT
+
+require 'fbe/octo'
+require 'fbe/iterate'
+require 'fbe/if_absent'
+require 'fbe/issue'
+
+start = Time.now
+
+events = %w[issue_type_added issue_type_changed]
+
+Fbe.iterate do
+  as 'types-were-scanned'
+  by "(agg (and (eq repository $repository) (eq what 'issue-was-opened') (gt issue $before)) (min issue))"
+  quota_aware
+  repeats 20
+  over do |repository, issue|
+    begin
+      Fbe.octo.issue_timeline(repository, issue).each do |te|
+        if Time.now - start > 5 * 60
+          $loog.debug("We are scanning labels for #{start.ago} already, it's time to quit")
```

Comment:
@Yegorov typo here? Not "labels" but "types"

Reply:
@yegor256 fixed!